app/controllers/users_controller.rb

```diff
@@ -0,0 +1,74 @@
+class UsersController < ApplicationController
+  STATES = ['AC','AL','AM','AP','BA','CE','DF','ES','GO','MA','MG','MS','MT','PA','PB','PE','PI','PR','RJ','RN','RS','RO','RR','SC','SE','SP','TO'].freeze
```

Comment:
Just for the next ones, when you start to use fudge, for a better CR you can work only with 112 columns